tests/test.py

```diff
@@ -0,0 +1,32 @@
+import os
+import pyChessBoardDetector
+
+def test_binding():
+    if(pyChessBoardDetector.test_bindings()== "Hello Binders!"):
+        assert pyChessBoardDetector.test_bindings() == "Hello Binders!"
+        print("Sucess")
+    else:
+        print("Fail")
+
+
+def test_cornerdetector():
+    print("Sucess From cornerdetector")
+    images_path = []
+    file_path = os.path.dirname(os.path.realpath(__file__))
+    image_path = os.path.join(file_path,"IMG_3715.jpg")
+    images_path.append(image_path)
+    image_path = os.path.join(file_path,"IMG_3134.jpg")
+    images_path.append(image_path)
+    # print("File Path:",image_path)
+    output = pyChessBoardDetector.get_chessboard_corners(images_path,9,7,False)
+    for path in images_path:
```

Comment:
dont print.
add assert statemens checking the dictionary key is matching with the filenames you have given